worker/src/main/scala/org/apache/celeborn/service/deploy/worker/Worker.scala

```diff
@@ -275,6 +276,12 @@ private[celeborn] class Worker(
   private val userResourceConsumptions =
     JavaUtils.newConcurrentHashMap[UserIdentifier, (ResourceConsumption, Long)]()
 
+  private var jvmQuake: JVMQuake = _
+  if (conf.workerJvmQuakeEnabled) {
+    jvmQuake = JVMQuake.create(conf, workerInfo.toUniqueId().replace(":", "-"))
+    jvmQuake.start()
+  }
```

Comment:
jvmQuake.start() also check conf.workerJvmQuakeEnabled，redundant

Reply:
@jiaoqingbo, I have already checked the `conf.workerJvmQuakeEnabled` in `JVMQuake#start`.

Reply:
@jiaoqingbo, the check in `JVMQuake#start` is check for `JVMQuake` itself. Meanwhile, the check in creator of `JVMQuake` is check to determine whether to create.